src/finn/transformation/fpgadataflow/specialize_layers.py

```diff
@@ -63,6 +63,11 @@ def _determine_impl_style(node):
                     return "rtl"
                 else:
                     return "hls"
+            elif optype == "VVAU":
```

Comment:
We should do here the same as we discussed for the MVAU, if the user has no preference, only convert to the RTL variant if it is the reasonable choice.